rapid7_insightidr/komand_rapid7_insightidr/actions/advanced_query_on_log_set/action.py

```diff
@@ -86,8 +69,13 @@ def get_results_from_callback(self, callback_url: str, timeout: int) -> [object]
         results_object = response.json()
         log_entries = results_object.get("events")
 
+        if results_object.get("links"):
+            callback_url = results_object.get("links")[0].get("href")
```

Comment:
Same concern here as the other instance I commented on

Reply:
If there's a links entry, it won't be null here.